From 2f03bdae6aa2655dcbf83bd3ab6c56234e585a70 Mon Sep 17 00:00:00 2001
From: Federico Mena Quintero <federico@gnome.org>
Date: Wed, 17 Nov 2021 13:55:13 -0600
Subject: [PATCH 1/1] Makefile.am: use the correct EXEEXT extension for the
 rsvg-convert binary

Copied from msys2 mingw-packages' patches.

Part-of: <https://gitlab.gnome.org/GNOME/librsvg/-/merge_requests/636>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index e93714e4..f05d0372 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -226,7 +226,7 @@ $(RSVG_CONVERT_BIN): $(RSVG_CONVERT_SRC)
 	$(CARGO) --locked build $(CARGO_VERBOSE) $(CARGO_TARGET_ARGS) $(CARGO_RELEASE_ARGS) --bin rsvg-convert
 
 rsvg-convert$(EXEEXT): $(RSVG_CONVERT_BIN)
-	cd $(LIBRSVG_BUILD_DIR) && mv $(RSVG_CONVERT_BIN) rsvg-convert
+	cd $(LIBRSVG_BUILD_DIR) && mv $(RSVG_CONVERT_BIN) rsvg-convert$(EXEEXT)
 
 dist_man_MANS = rsvg-convert.1
 
-- 
2.33.1

